### PR TITLE
Add 3 tasks for Vietnamese

### PR DESCRIPTION
**Pull request by taidnguyen**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

Add ViQuAD (retrieval), StudentFeedback (sentiment classification), and MedEV (bitext mining) for Vietnamese.

In my view, they are high-quality datasets with reasonable performances. Hoping to potentially add more Viet datasets in the near future.

@KennethEnevoldsen would appreciate your help to make sure things look good and with point calculation. Thanks!

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

File: mteb/tasks/BitextMining/vie/VieMedEVBitextMining.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+import datasets
+import random
+from mteb.abstasks import AbsTaskBitextMining
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+TEST_SAMPLES = 2048
+
+class VieMedEVBitextMining(AbsTaskBitextMining):
```
Comment (KennethEnevoldsen) [inline mteb/tasks/BitextMining/vie/VieMedEVBitextMining.py:10]: 2 points for new dataset

File: mteb/tasks/Classification/vie/VieStudentFeedbackClassification.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import random
+from collections import Counter
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+TEST_SAMPLES = 2048
+
+
+class VieStudentFeedbackClassification(AbsTaskClassification):
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/vie/VieStudentFeedbackClassification.py:12]: 2 points for new dataset (+4 bonus for first classification task for the language). We do have a vie task in MassiveIntent, but it is a translation so I would not count it.

File: mteb/tasks/Retrieval/vie/VieQuADRetrieval.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+from datasets import load_dataset
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+TEST_SAMPLES = 2048
+
+
+class VieQuADRetrieval(AbsTaskRetrieval):
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Retrieval/vie/VieQuADRetrieval.py:14]: 2 +4 here as well.

File: mteb/tasks/BitextMining/vie/VieMedEVBitextMining.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        eval_langs={
+            "en-vie": ["eng-Latn", "vie-Latn"],
+            "vie-en": ["vie-Latn", "eng-Latn"],
+        },
+        main_score="f1",
+        date=("2024-08-28", "2022-03-28"),
+        form=["written"],
+        domains=["Medical"],
+        task_subtypes=[],
+        license=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/BitextMining/vie/VieMedEVBitextMining.py:32]: required

File: mteb/tasks/Retrieval/vie/VieQuADRetrieval.py [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["validation"],
+        eval_langs=["vie-Latn"],
+        main_score="ndcg_at_10",
+        date=("2022-03-02", "2022-03-02"),
+        form=["written"],
+        domains=["Encyclopaedic", "Non-fiction"],
+        task_subtypes=["Question answering"],
+        license=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Retrieval/vie/VieQuADRetrieval.py:32]: required

**Review by KennethEnevoldsen — APPROVED** [review]
Wonderful additions and almost perfect metadata. I have added some comments on the points will you also add 1 point for the review as well (to my name)

**Comment by taidnguyen** [comment]
Thanks @KennethEnevoldsen ! Updated missing metadata and points

**Comment by KennethEnevoldsen** [comment]
Wonderful @taidnguyen, I have set the PR to auto-merge once the tests pass

**Pull request merged**